Jenkinsfile

```diff
@@ -58,6 +58,7 @@ pipeline {
         AWS = credentials('jenkins-aws')
         STORAGE_USR="${AWS_USR}"
         STORAGE_PSW="${AWS_PSW}"
+        MAIN_AUDIENCE = credentials('main-audience')
```

Comment:
the audience is not a secret, I think it might be a committed constant in the pipeline

Reply:
I use a Global Var in Jenkins.
Is it acceptable in your opinion?

Reply:
yup, works for me